ajax/libs/togeojson/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "togeojson",
+  "filename": "togeojson.min.js",
+  "description": "convert KML and GPX to GeoJSON",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mapbox/togeojson.git"
+  },
+  "license": "BSD-2-Clause",
+  "keywords": [
+    "kml",
+    "geojson",
+    "gpx",
+    "geo",
+    "parser",
+    "formatter",
+    "input",
+    "leaflet",
+    "maps"
+  ],
+  "homepage": "https://mapbox.github.io/togeojson/",
+  "author": "tmcw",
```

Comment:
He is the main contributor of the library https://github.com/mapbox/togeojson/graphs/contributors